src/main/java/Action.java

```diff
@@ -0,0 +1,150 @@
+                    break;
+                }
+            case "help":
+                ArrayList<Item> playerItems = player.getInventory();
+                String playerInventory = "";
+                String playerInventoryFinal;
+                if (playerItems.size() != 0){
+                    playerInventoryFinal = printInventory(player.getInventory());
+                } else {
+                    playerInventoryFinal = "Your current inventory is empty";
+                }
+                System.out.println(Main.ANSI_BLUE + "\nYou are currently located in " + player.getCurrentRoom().getRoomName() +
+                        ".\n" + playerInventoryFinal + ".\n\nTo move, type " + Main.ANSI_MAGENTA +  "'move to <location>'" +
+                        Main.ANSI_BLUE +  ".\nTo see where to move to next and what items are in this location, type " +
+                        Main.ANSI_MAGENTA + "'look around'" + Main.ANSI_BLUE + ".\nTo add an item to your inventory, type " +
+                        Main.ANSI_MAGENTA +  "'pick up <item>'" + Main.ANSI_BLUE + ".\nTo learn more about an item, type " +
+                        Main.ANSI_MAGENTA + "'inspect <item>'" + Main.ANSI_BLUE + ".\nTo quit the game, type " +
+                        Main.ANSI_MAGENTA + "'quit'" + Main.ANSI_BLUE + ".\n" + Main.ANSI_RESET);
+
+                break;
+            case "look":
+                if (command[1].equals("around") && command.length == 2){
+                    String[] roomOptions = player.getCurrentRoom().getNextRooms();
+                    String roomOptionsString = roomOptions[0];
+                    for (int i = 1; i < roomOptions.length; i++){
+                        roomOptionsString = roomOptions[i] + ", " + roomOptionsString;
+                    }
+                    HashMap<String, Item> roomItems = player.getCurrentRoom().getItems();
+                    String itemOptionsString = "";
+                    String itemOptionsFinalString;
+                    if (roomItems.isEmpty()){
+                        itemOptionsFinalString = "none";
+                    } else {
+                        for (String itemName : roomItems.keySet()){
+                            itemOptionsString = itemName + ", " + itemOptionsString;
+                        }
+                        int unnecessaryComma1 = itemOptionsString.lastIndexOf(",");
+                        itemOptionsFinalString = itemOptionsString.substring(0, unnecessaryComma1);
+                    }
+                    System.out.println(Main.ANSI_BLUE + "\nYou can move to " + roomOptionsString + ".\nItem(s) located" +
+                            "in this room: " + itemOptionsFinalString + ".\n" + Main.ANSI_RESET);
+                    break;
+                }
+
+            default:
+                System.out.println(Main.ANSI_BLUE + "\nThe command name is invalid. Please ask for help.\n" + Main.ANSI_RESET);
+        }
+    }
+}
```

Comment:
It works, but you have a gigantic switch statement with deeply nested conditionals. Polymorphism isn't used much and a lot of classes are heavily referenced by each other, changes in one would necessitate changes across the board.